plugins/SedRegex/plugin.py

```diff
@@ -239,8 +235,15 @@ def _replacer_process(self, irc, msg, target, pattern, replacement, count, messa
 
                         subst = axe_spaces(subst)
 
-                        return _("%s meant to say: %s") % \
-                            (messageprefix, subst)
+                        if m.nick == msg.nick:
+                            fmt = self.registryValue('sedString', msg.channel, irc.network)
+                            env = {'nick': m.nick, 'replacement': subst}
+                        else:
+                            fmt = self.registryValue('sedEdited', msg.channel, irc.network)
+                            env = {'nick': m.nick, 'otherNick': msg.nick, 'replacement': subst}
+
+                        return ircutils.standardSubstitute(irc, msg, fmt, env)
```

Comment:
```suggestion
                        if m.nick == msg.nick:
                            fmt = self.registryValue('sedString', msg.channel, irc.network)
                            env = {'replacement': subst}
                        else:
                            fmt = self.registryValue('sedEdited', msg.channel, irc.network)
                            env = {'otherNick': msg.nick, 'replacement': subst}

                        return ircutils.standardSubstitute(irc, m, fmt, env)
```

`standardSubstitute` will automatically set `$nick` from the `m` message

Reply:
Oh okay, didn't know that, I'll change that.